test/elixir_google_scraper_web/controllers/user_session_controller_test.exs

```diff
@@ -11,9 +11,9 @@ defmodule ElixirGoogleScraperWeb.UserSessionControllerTest do
     test "renders log in page", %{conn: conn} do
       conn = get(conn, Routes.user_session_path(conn, :new))
       response = html_response(conn, 200)
-      assert response =~ "<h1>Log in</h1>"
-      assert response =~ "Log in</a>"
-      assert response =~ "Register</a>"
+      assert response =~ "Log in"
+      assert response =~ "Log in"
```

Comment:
I think these two is the same check, it will probably match the first one, maybe we can just assert it once and rely on the feature tests?

```suggestion
      assert response =~ "Log in"
```